packages/twenty-server/src/workspace/workspace-health/services/database-structure.service.ts

```diff
@@ -122,17 +125,17 @@ export class DatabaseStructureService {
     }));
   }
 
-  getPostgresDataType(
-    fieldMetadataType: FieldMetadataType,
-    fieldMetadataName: string,
-    objectMetadataNameSingular: string,
-  ): string {
-    const typeORMType = fieldMetadataTypeToColumnType(fieldMetadataType);
+  getPostgresDataType(fieldMetadata: FieldMetadataEntity): string {
+    const typeORMType = fieldMetadataTypeToColumnType(fieldMetadata.type);
     const mainDataSource = this.typeORMService.getMainDataSource();
 
-    // TODO: remove special case for enum type, should we include this to fieldMetadataTypeToColumnType?
```

Comment:
Actually `fieldMetadataTypeToColumnType` is used in other places, so we can't change it.
This special case only happen here